pkg/testutil/pointers.go

```diff
@@ -0,0 +1,6 @@
+package testutil
+
+// NullInt returns a pointer to the given int.
+func NullInt(i int) *int {
```

Comment:
Do we need this or could we just use go-fastly's `ToPointer()` method?

Reply:
Oh I missed that, thank you!